src/templates/Plugin.js

```diff
@@ -11,11 +11,34 @@ import {
   Title,
   TextLink as Link,
 } from 'components';
-import { Header, PluginCTA, PlaceholderBody, Sidebar } from 'components/backstage/plugins';
+import { Header, PluginCTA, Sidebar } from 'components/backstage/plugins';
 import fullRoadieDocsPath from 'components/backstage/plugins/fullRoadieDocsPath';
 import PluginFeedbackModal from '../components/PluginFeedbackModal';
 import useScrollToElement from '../hooks/useScrollToElement';
 
+const PAGE_SECTIONS = {
+  INTRODUCTION: {
+    key: 'introduction',
+    fragment: 'introduction',
+    label: 'Introduction',
+  },
+  INSTALLATION_INSTRUCTIONS: {
+    key: 'installationInstructions',
+    fragment: 'installation-instructions',
+    label: 'Installation Instructions',
+  },
+  NOTES: {
+    key: 'notes',
+    fragment: 'notes',
+    label: 'Things to Know',
+  },
+  CHANGELOG: {
+    key: 'changelog',
+    fragment: 'changelog',
+    label: 'Changelog',
+  },
+};
+
```

Comment:
Pulled these out into a constants because they're referred to from a few places and I don't want the headings & fragments to get out of sync.